src/components/CalendarMonthGrid.jsx

```diff
@@ -363,7 +367,14 @@ class CalendarMonthGrid extends React.Component {
 CalendarMonthGrid.propTypes = propTypes;
 CalendarMonthGrid.defaultProps = defaultProps;
 
-export default withStyles(({ reactDates: { color, zIndex } }) => ({
+export default withStyles(({
+  reactDates: {
+    color,
+    noScrollBarOnVerticalScrollable,
```

Comment:
QQ: can you investigate as to whether the scrollbar is an issue on the `VERTICAL_ORIENTATION` as well? It may make sense to have `noScrollBarOnVertical` be the option

Reply:
It isn't/if there were an issue, it could be managed outside of the vertical calendar (because the largest div for the vertical calendar is the outermost div, whereas the smaller surrounding div in VERTICAL_SCROLLABLE is the one that's causing the scrollbar to appear).